sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/codegen/CodeGenerator.scala

```diff
@@ -1318,10 +1332,11 @@ object CodeGenerator extends Logging {
   }
 
   /**
-   * Returns the max bytecode size of the generated functions by inspecting janino private fields.
-   * Also, this method updates the metrics information.
+   * Returns the bytecode statistics (max class bytecode size, max method bytecode size, and
+   * max constant pool size) of generated classes by inspecting janino private fields.inspecting
```

Comment:
Nit: `inspecting janino private fields.inspecting janino private fields` seems weird.
Also: could we always spell "Janino" as such?

Reply:
oh... I''ll fix soon. Thanks!